### ISSUE TITLE
FBCapture SDK 1.2 source code available?

### ISSUE THREAD

**Issue by tvkamara** [facebook/360-Capture-SDK#36]
Hi and thanks for the create SDK,

I was wondering are you planning on publishing the source code for the 1.2 update of the FBCapture SDK. Currently only the built dll is available. I'm using the old FBCapture (open source) in my project with slight modifications, however I'm interested in the performance updates of the latest edition.

Thanks,
-Teemu

**Comment by cg439** [comment]
Yes, we'll update the source for building it as well though it may take an
extra week or two

**Comment by whitespeed** [comment]
+1 :) look forward to source code!

**Comment by andreydobrikov** [comment]
@cg439 has there been an update on this? 1.3.0 has been released but not the source code for native part of FBCapture SDK. I have my own modifications to the native part of code and would like to get the updates to merge with my changes.

**Comment by whitespeed** [comment]
Now it's 2.0. Where is the native source code ?
